sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/pom.xml

```diff
@@ -39,7 +35,7 @@
         <dependency>
             <groupId>com.alibaba.nacos</groupId>
             <artifactId>nacos-client</artifactId>
-            <version>${nacos-client.verison}</version>
+            <version>${nacos.client.verison}</version>
             <scope>provided</scope>
```

Comment:
Why is provided?

Reply:
nacos is not a default implemention, it provides for user selection, so it is provided.